gamestonk_terminal/cryptocurrency/coinpaprika/coinpaprika_view.py

```diff
@@ -0,0 +1,1541 @@
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def all_platforms(other_args: List[str]):
+    """List all smart contract platforms like ethereum, solana, cosmos, polkadot, kusama
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="platforms",
+        add_help=False,
+        description="""List all smart contract platforms like ethereum, solana, cosmos, polkadot, kusama
+        """,
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = paprika.get_all_contract_platforms()
+        print(
+            tabulate(
+                df,
+                headers=df.columns,
+                floatfmt=".0f",
+                showindex=False,
+                tablefmt="fancy_grid",
+            )
+        )
+        print("")
+
+    except Exception as e:
+        print(e)
+        print("")
+
+
+def contracts(other_args: List[str]):
+    """Gets all contract addresses for given platform
+
+    Parameters
+    ----------
+    other_args: List[str]
+        Arguments to pass to argparse
+
+    """
+    parser = argparse.ArgumentParser(
+        prog="contracts",
+        add_help=False,
+        description="""Gets all contract addresses for given platform.
+        Provide platform id with -p/--platform parameter
+        You can display only top N number of smart contracts with --top parameter.
+        You can sort data by index, id, type, active, address  --sort parameter
+        and also with --descend flag to sort descending.
+
+        Displays:
+            index, id, type, active, address
+        """,
+    )
+
+    parser.add_argument(
+        "-p",
+        "--platform",
+        help="Blockchain platform like eth-ethereum",
+        dest="platform",
+        default="eth-ethereum",
+        type=str,
+        choices=PLATFORMS,
+    )
+
+    parser.add_argument(
+        "-t", "--top", default=20, dest="top", help="Limit of records", type=int
+    )
+
+    parser.add_argument(
+        "-s",
+        "--sort",
+        dest="sortby",
+        type=str,
+        help="Sort by given column. Default: index",
+        default="index",
+        choices=["index", "id", "type", "active", "address"],
+    )
+
+    parser.add_argument(
+        "--descend",
+        action="store_false",
+        help="Flag to sort in descending order (lowest first)",
+        dest="descend",
+        default=True,
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+
+        df = paprika.get_contract_platform(ns_parser.platform)
+
+        if df.empty:
+            print(f"Nothing found for platform: {ns_parser.platform}")
```

Comment:
Missing the new line.

One way to add a new line in a print, that I found out recently and I quite like is:
```
print(f"Nothing found for platform: {ns_parser.platform}", "\n")
```